app/screens/onboarding/paginator.tsx

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {View, useWindowDimensions, TouchableOpacity} from 'react-native';
+import Animated, {interpolate, useAnimatedStyle} from 'react-native-reanimated';
+
+import {makeStyleSheetFromTheme} from '@utils/theme';
+
+type Props = {
+    dataLength: number;
+    theme: Theme;
+    scrollX: Animated.SharedValue<number>;
+    moveToSlide: (slideIndexToMove: number) => void;
+};
+
+const DOT_SIZE = 16;
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    dot: {
```

Comment:
0/5 I see a lot of repetition here, so we can do this in a slight different way:

```
const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
    const commonStyle = {
        height: DOT_SIZE / 2,
        borderRadius: 5,
        backgroundColor: theme.buttonBg,
        marginHorizontal: DOT_SIZE / 2,
        width: DOT_SIZE / 2,
        opacity: 0.25,
    };
    return {
        dot: {
            ...commonStyle,
        },
        fixedDot: {
            ...commonStyle,
            position: 'absolute',
         ...
    }
});
```